lib/langchain/utils/token_length_validator.rb

```diff
@@ -51,13 +51,14 @@ class TokenLengthValidator
       #
       # Calculate the `max_tokens:` parameter to be set by calculating the context length of the text minus the prompt length
       #
-      # @param text [String] The text to validate
+      # @param content [String | Array<String>] The text or array of texts to validate
       # @param model_name [String] The model name to validate against
       # @return [Integer] Whether the text is valid or not
       # @raise [TokenLimitExceeded] If the text is too long
       #
-      def self.validate_max_tokens!(text, model_name)
-        text_token_length = token_length(text, model_name)
+      def self.validate_max_tokens!(content, model_name)
+        text_token_length = Array(content).sum { |text| token_length(text, model_name) }
```

Comment:
Are there any other use-cases where `validate_max_tokens!()` would be called with content being an array?